tests/server/server-executeCommand-002.phpt

```diff
@@ -0,0 +1,78 @@
+--TEST--
+MongoDB\Driver\Server::executeCommand() takes a read preference
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("REPLICASET"); ?>
```

Comment:
I'm not sure how it happened, but I somehow stopped using `__DIR__ . "/" ."../utils/basic-skipif.inc";` and regret it a lot now.

Having the leading `/` in quotes of its own allows the second component to be fully qualified filename relative to the cwd.
That means all editors will be able to jump to that file trivially. As it is now, the leading `/` makes it into nonexisting absolute path :(